tests/integration/image-builders/operator/catalogs/catalog.Dockerfile

```
@@ -1,11 +1,17 @@
-ARG CATALOG="catalog"
+FROM quay.io/operator-framework/opm:latest as builder
+ARG CATALOG
 
-FROM quay.io/operator-framework/opm:latest
+# Copy FBC root into image at /configs and pre-populate serve cache
+ADD $CATALOG /configs/
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)